bentoml/yatai/deployment/aws_lambda/lambda_app.py

```diff
@@ -14,13 +14,17 @@
 
 import os
 import sys
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
after `import bentoml` below, the logging format and level might get reset, so probably don't need this `setLevel` here.

Could you move `logger = logging.getLogger(__name__)` to after the `from bentoml import load` line?  and keep the logging message in `download_extra_resources` just using `print`?

It is a bit tricky here because `bentoml` package may not be available to import before the `download_extra_resources` call, as the package file might be stored in s3 and needs to be downloaded first.